### DISCUSSION TITLE
Alert not ever getting set?

### DISCUSSION THREAD

**Discussion by agillesp** [datalust/seq-tickets#1539] [Help]
I've setup an alert pretty much unchanged from examples: any amount of errors within `1m`. The query returns the correct data from the stream, the chart shows "hits", but the alert is never set. The chart is always ✅ and I never receive email alerts. Is this a bug for the latest version?

<img width="281" alt="Screenshot 2022-04-02 073043" src="https://user-images.githubusercontent.com/1098950/161387948-0ad2486f-79ef-433b-bf34-b23fafa8d4e5.png">

**Comment by liammclennan** [comment]
That looks like the query returns results, but does not exceed 'the condition that will trigger the alert'. If you can send more detail of your alert configuration to `support@datalust.co` we will try to suggest a fix.

**Reply by agillesp** [reply]
I thought that was the `having > 0` bits of the query? `The condition that will trigger the alert [...]` according to the description. Here's a snap of what I've got.

<img width="1042" alt="Screenshot 2022-04-03 171504" src="https://user-images.githubusercontent.com/1098950/161455326-28e83858-8b10-44f0-893a-a9820a0870c3.png">

**Reply by liammclennan** [reply]
Is it possible that the query has changed? Otherwise it should not be possible to have results that don't trigger that query?

What do you get when you run the test?

**Reply by agillesp** [reply]
It's never changed. This is what I ran with out the gate. The dashboard continues to show as in the first screen grab. But for completeness, here is the query run as I've typed this ...

<img width="999" alt="Screenshot 2022-04-03 194722" src="https://user-images.githubusercontent.com/1098950/161465738-9ee30594-d282-43f2-9484-b3e237057c38.png">

Full disclosure - I did have to back the date out minus one day, only because that's last when I was testing the system. But otherwise everything's unchanged.

**Reply by liammclennan** [reply]
Can you try sending an event with @Level = 'Error'? [seqcli](https://docs.datalust.co/docs/command-line-client) can do this. Something like:

```shell
> seqcli log -l Error -m "Testing error alert" -s <server url> -a <apikey>
```

**Reply by agillesp** [reply]
Mind giving me a curl / Postman version of this? I've got a Docker image running in AWS that's hosting this.

**Reply by agillesp** [reply]
N/m, sorry, I thought it required the full install of Seq. I'll install the CLI and report back. Thanks.

**Reply by agillesp** [reply]
Well that absolutely worked. 😵 Alarm set + emailed.

**Reply by agillesp** [reply]
And the exact same test that wasn't working yesterday is suddenly working. I swear I didn't change anything. Jeez. Thanks for the help. I guess we can consider this resolved.

<img width="532" alt="Screenshot 2022-04-03 205507" src="https://user-images.githubusercontent.com/1098950/161471333-ddb595da-611d-40e3-a58b-5ef86dc81153.png">

**Reply by liammclennan** [reply]
That's great news. Glad we could help.
